Discussion/DiscussionTests/Presentation/Comment/Base/BaseResponsesViewModelTests.swift

```diff
@@ -343,7 +408,12 @@ final class BaseResponsesViewModelTests: XCTestCase {
         let interactor = DiscussionInteractorProtocolMock()
         let router = DiscussionRouterMock()
         let config = ConfigMock()
-        let viewModel = BaseResponsesViewModel(interactor: interactor, router: router, config: config)
+        let viewModel = BaseResponsesViewModel(
```

Comment:
```
        let interactor = DiscussionInteractorProtocolMock()
        let router = DiscussionRouterMock()
        let config = ConfigMock()
        let viewModel = BaseResponsesViewModel(
            interactor: interactor,
            router: router,
            config: config,
            storage: CoreStorageMock()
        )
```

These properties are using in multiple methods so we should define them as instance properties of the test class and initialize them in the `setUp` method.

e.g

`override func setUp() {
        super.setUp()
        
        interactor = DiscussionInteractorProtocolMock()
        router = DiscussionRouterMock()
        config = ConfigMock()
        viewModel = BaseResponsesViewModel(
            interactor: interactor,
            router: router,
            config: config,
            storage: CoreStorageMock()
        )
    }`